LICENSE

```diff
@@ -1,21 +0,0 @@
-MIT License
```

Comment:
WTFPL IS GARBAGE PLEASE SWITCH TO UNLICENSE THANKS

Reply:
UNLICENSE IS GARBAGE PLEASE SWITCH TO AGPL THANKS